### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by bcanseco**
Opening PR for @kaufmann42

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -23,18 +21,30 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+  <title>Issueist</title>
+  <!-- Hotjar Tracking Code for https://issueist.wolfpak.com -->
```
Comment (bcanseco) [inline public/index.html]: Incorrect URL

File: public/index.html [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
-    />
-    <meta name="theme-color" content="#000000" />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="theme-color" content="#000000" />
```
Comment (bcanseco) [inline public/index.html]: This theme color should be one of Issueist's colors (either the lightning bolt from the logo or the teal-ish background)

Comment (bcanseco) [reply public/index.html]: Speaking of which - shouldn't the app itself match the Issueist colors? Specifically the app bar background

image

File: website/pages/en/index.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ class HomeSplash extends React.Component {
               <FaMagic size="3em" />
               <h4>Run From Anywhere</h4>
               <p>
-                Issueist is a PWA, giving it incredible flexibility. Download the chrome extension, or just embed <a>https://issueist.wolfpak.now.sh</a> anywhere in your browser to use it.
+                Issueist is a PWA, giving it incredible flexibility. Download the chrome extension, or just embed <a href="https://issueist.wolfpak.now.sh" rel="noopener noreferrer" target="_blank">https://issueist.wolfpak.now.sh</a> anywhere in your browser to use it.
```
Comment (bcanseco) [inline website/pages/en/index.js]: https://github.com/kaufmann42/issueist/blob/final-touches/public/manifest.json

The `start_url` in the PWA manifest should probably point to https://issueist.wolfpak.now.sh

Also iirc to *actually* be a PWA, you need to enable a service worker. We need to actually test it out on mobile and see if we get the prompt to download

File: public/index.html (hunk 1 above)
Comment (kaufmann42) [reply public/index.html]: fixed

**Pull request merged**